cypress/plugins/index.ts

```diff
@@ -9,12 +9,14 @@
 // https://on.cypress.io/plugins-guide
 // ***********************************************************
 
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import happoTask from 'happo-cypress/task'
+
 // This function is called when a project is opened or re-opened (e.g. due to
 // the project's config changing)
-
-const pluginConfig: Cypress.PluginConfig = () => {
-  // `on` is used to hook into various events Cypress emits
-  // `config` is the resolved Cypress config
+const pluginConfig: Cypress.PluginConfig = (on) => {
```

Comment:
```suggestion
  // `on` is used to hook into various events Cypress emits
const pluginConfig: Cypress.PluginConfig = (on) => {
```

Might be nice to leave the comment in?

Reply:
Sure, let me get the original comment back.